CONTRIBUTING.MD

```diff
@@ -0,0 +1,109 @@
+
+If you are a member of the `dbt Labs` GitHub organization, you will have push access to the `dbt-snowflake` repo. Rather than forking `dbt-snowflake` to make your changes, just clone the repository, check out a new branch, and push directly to that branch.
+
+## Running `dbt-snowflake` in development
+
+### Installation
+
+1. First make sure you have the latest version of `pip` installed by running `pip install --upgrade pip` in the terminal. 
+
+2. Configure and activate a `virtualenv` as described in [Setting up an environment](https://github.com/dbt-labs/dbt-core/blob/HEAD/CONTRIBUTING.md#setting-up-an-environment)
+
+3. Install `dbt-core`. To confirm you are using the correct version installed run `dbt --version` and `which dbt`
+
+4. Install `dbt-snowflake` and dependencies by running `pip install -e . -r dev-requirements.txt` in the same `virtual env`.  
+
+When `dbt-snowflake` is installed this way, any changes you make to the `dbt-snowflake` source code will be reflected immediately in your next `dbt-snowflake` run.
+
+## Testing
+
+### Initial Setup
+
+`dbt-snowflake` contains [unit](https://github.com/dbt-labs/dbt-snowflake/tree/main/tests/unit) and [integration](https://github.com/dbt-labs/dbt-snowflake/tree/main/tests/integration) tests. Integration tests require testing against an actual Snowflake warehouse. There are two major ways to run tests. 
+
+- This repo has a CI/CD GithubAction to run tests against a Snowflake warehouse during PR checks.
+
+- You can also run the integration tests locally by configuring a `test.env` file with the approprate `env` variables which you can obtain from `test.env.example` and corresponding credentials from your valid Snowflake account.
+
+Note: This `test.env` file is git-ignored, but please be _extra_ careful to never check in credentials or other sensitive information when developing. To create your `test.env`.
+
+```
+cp test.env.example test.env
+$EDITOR test.env
+```
+
+### Testing Locally Commands
+There are a few methods for running tests locally.
+
+#### `tox`
+`tox` takes care of managing Python virtualenvs and installing dependencies in order to run tests. You can also run tests in parallel, for example you can run unit tests for Python 3.7, Python 3.8, Python 3.9, and `flake8` checks in parallel with `tox -p`. Also, you can run unit tests for specific python versions with `tox -e py37`. The configuration of these tests are located in `tox.ini`.
+
+#### `pytest`
+You can also run a specific test or group of tests using `pytest` directly. With a Python virtualenv active and dev dependencies installed you can run test commands like:
+
+```sh
+# run specific snowflake integration tests
+python -m pytest -m profile_snowflake tests/integration/simple_copy_test
+# run all unit tests in a file
+python -m pytest tests/unit/test_snowflake_adapter.py
+# run a specific unit test
+python -m pytest tests/unit/test_snowflake_adapter.py::TestSnowflakeAdapter::test_convert_date_type
```

Comment:
also add an integration test example or just say this can either be unit or integration